src/ripple/protocol/impl/SecretKey.cpp

```diff
@@ -33,19 +33,19 @@ namespace ripple {
 
 SecretKey::~SecretKey()
 {
-    beast::secure_erase(buf_, sizeof(buf_));
+    beast::secure_erase(buf_.data(), buf_.size());
 }
 
 SecretKey::SecretKey (std::array<std::uint8_t, 32> const& key)
 {
-    std::memcpy(buf_, key.data(), key.size());
+    std::memcpy(buf_.data(), key.data(), key.size());
```

Comment:
Why not simply:

    SecretKey::SecretKey (std::array<std::uint8_t, 32> const& key)
        : buf_(key)
    {
    }

Reply:
Addressed in 4ecdd58c54c23ffbb297be83b8252e53631287fe